### PR TITLE
DOCS-2382: Add and update CWS rules

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds a section dedicated to adding and updating CWS rules to the existing CWS rules docs

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/cws-rules/security_platform/cloud_workload_security/workload_security_rules

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 1]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
 
 Approvers are generally more powerful and preferred. Using approvers, the Agent can process only what it needs to see rather than dynamically learning what to filter out.
 
+## Add and update rules
+
+To receive the latest bundled CWS rules, [upgrade the Datadog Agent][3].
+
+To manually add or update rules, load a new `default.policy` file to the Agent. Follow the instructions below:
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:75]:
```suggestion
To manually add or update rules, load a new `default.policy` file to the Agent. 

Follow the instructions below:
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 2]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+    configMap: *jdefaultpol*
+      syscallMonitor:
+        # datadog.securityAgent.runtime.syscallMonitor.enabled
+        # Set to true to enable Syscall monitoring.
+        enabled: false
+    ```
+
+10. Upgrade the helm chart with the following command: `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`
+
+    **Note:** If you need to make further changes to `default.policy` you can either use `kubectl edit cm jdefaultpol` or replace the configMap with  `kubectl create configmap jdefaultpol --from-file default.policy -o yaml --dry-run=client | kubectl replace -f -`
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:136]:
```suggestion
    **Note:** If you need to make further changes to `default.policy`, you can either use `kubectl edit cm jdefaultpol` or replace the configMap with  `kubectl create configmap jdefaultpol --from-file default.policy -o yaml --dry-run=client | kubectl replace -f -`.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 3]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+
+[1]: /agent/guide/agent-commands/?tab=agentv6v7#restart-the-agent
+{{% /tab %}}
+
+{{% tab "Helm" %}}
+
+8. Create a configmap containing `default.policy`. For example, `kubectl create configmap jdefaultpol --from-file=default.policy`.
+
+9. Add the configMap (`jdefaultpol`) into values.yaml, `datadog.securityAgent.runtime.policies.configMap`:
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:114]:
```suggestion
9. Add the configMap (`jdefaultpol`) into `values.yaml` with `datadog.securityAgent.runtime.policies.configMap`:
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 4]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+    {{< img src="security_platform/cws/workload_security_rules/define_runtime_expression.png" alt="Adding a rule to the expression field" >}}
+
+5. Define the logic for when this rule triggers a security signal. For example, `a>0` means a security signal triggers as long as the rule condition set in step 4 is met at least once in the sliding time window. Select a severity to associate the rule with and select all relevant parties you to notify.
+
+    {{< img src="security_platform/cws/workload_security_rules/rule_cases.png" alt="Setting a rule trigger, severity, and notification" >}}
+
+6. Name the rule and add the actual notification message in markdown format. It is recommended to include a remediation runbook in the body. As noted in the template, use substitution variables to dynamically generate contextualized content at runtime. After the message, add multiple tags to give more context to the signals generated by your custom rule.
+
+7. Save the rule. This automatically navigates you back to the main Rules page. Click the **Download Workload Security Policy** button in the top right corner to download a `default.policy` file to your local machine.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:93]:
```suggestion
7. Save the rule. This automatically navigates you back to the main **Rules** page. Click **Download Workload Security Policy** in the top right corner to download a `default.policy` file to your local machine.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 5]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+4. Enter the rule in the Expression field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the *counter*, *group by* and *roll-up* window.
+
+    {{< img src="security_platform/cws/workload_security_rules/define_runtime_expression.png" alt="Adding a rule to the expression field" >}}
+
+5. Define the logic for when this rule triggers a security signal. For example, `a>0` means a security signal triggers as long as the rule condition set in step 4 is met at least once in the sliding time window. Select a severity to associate the rule with and select all relevant parties you to notify.
+
+    {{< img src="security_platform/cws/workload_security_rules/rule_cases.png" alt="Setting a rule trigger, severity, and notification" >}}
+
+6. Name the rule and add the actual notification message in markdown format. It is recommended to include a remediation runbook in the body. As noted in the template, use substitution variables to dynamically generate contextualized content at runtime. After the message, add multiple tags to give more context to the signals generated by your custom rule.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:91]:
```suggestion
6. Name the rule and add the actual notification message in Markdown format. Datadog recommends including a remediation runbook in the body. As noted in the template, use substitution variables to dynamically generate contextualized content at runtime. After the message, add multiple tags to give more context to the signals generated by your custom rule.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 6]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+
+To manually add or update rules, load a new `default.policy` file to the Agent. Follow the instructions below:
+
+1. Navigate to the [Rules page][4] in Datadog.
+2. Click **New Rule** button in the top right.
+3. Select **Workload Security** under rule types.
+
+    {{< img src="security_platform/cws/workload_security_rules/create_cws_rule.png" alt="Configuring a new Cloud Workload Security rule with rule type as Workload Security" >}}
+
+4. Enter the rule in the Expression field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the *counter*, *group by* and *roll-up* window.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:83]:
```suggestion
4. Enter the rule in the **Expression** field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the **counter**, **group by**, and **roll-up** window.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 7]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
 
 Approvers are generally more powerful and preferred. Using approvers, the Agent can process only what it needs to see rather than dynamically learning what to filter out.
 
+## Add and update rules
+
+To receive the latest bundled CWS rules, [upgrade the Datadog Agent][3].
+
+To manually add or update rules, load a new `default.policy` file to the Agent. Follow the instructions below:
+
+1. Navigate to the [Rules page][4] in Datadog.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:77]:
```suggestion
1. In Datadog, navigate to the [**Rules** page][4].
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 8]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+2. Click **New Rule** button in the top right.
+3. Select **Workload Security** under rule types.
+
+    {{< img src="security_platform/cws/workload_security_rules/create_cws_rule.png" alt="Configuring a new Cloud Workload Security rule with rule type as Workload Security" >}}
+
+4. Enter the rule in the Expression field using [Datadog Security Language (SECL)][5]. A rule can have multiple Agent expressions combined with boolean logic (`||`, `&&`). You can also change the *counter*, *group by* and *roll-up* window.
+
+    {{< img src="security_platform/cws/workload_security_rules/define_runtime_expression.png" alt="Adding a rule to the expression field" >}}
+
+5. Define the logic for when this rule triggers a security signal. For example, `a>0` means a security signal triggers as long as the rule condition set in step 4 is met at least once in the sliding time window. Select a severity to associate the rule with and select all relevant parties you to notify.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:87]:
```suggestion
5. Define the logic for when this rule triggers a security signal. For example, `a>0` means a security signal triggers as long as the rule condition set in Step 4 is met at least once in the sliding time window. Select a severity to associate the rule with and select all relevant parties you want to notify.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 9]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+          # datadog.securityAgent.runtime.policies.configMap
+          # Place custom policies here
+    configMap: *jdefaultpol*
+      syscallMonitor:
+        # datadog.securityAgent.runtime.syscallMonitor.enabled
+        # Set to true to enable Syscall monitoring.
+        enabled: false
+    ```
+
+10. Upgrade the helm chart with the following command: `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:134]:
```suggestion
10. Upgrade the helm chart with `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 10]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+6. Name the rule and add the actual notification message in markdown format. It is recommended to include a remediation runbook in the body. As noted in the template, use substitution variables to dynamically generate contextualized content at runtime. After the message, add multiple tags to give more context to the signals generated by your custom rule.
+
+7. Save the rule. This automatically navigates you back to the main Rules page. Click the **Download Workload Security Policy** button in the top right corner to download a `default.policy` file to your local machine.
+
+Complete the remaining steps based on your environment:
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+8. Copy the `default.policy` file over to the target host, in the `{$DD_AGENT}/runtime-security.d` folder. Ensure the file minimally has `read` and `write` access for the `dd-agent` user on the host.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:100]:
```suggestion
8. Copy the `default.policy` file over to the target host in the `{$DD_AGENT}/runtime-security.d` folder. Ensure the file minimally has `read` and `write` access for the `dd-agent` user on the host.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 11]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
 Approvers are generally more powerful and preferred. Using approvers, the Agent can process only what it needs to see rather than dynamically learning what to filter out.
 
+## Add and update rules
+
+To receive the latest bundled CWS rules, [upgrade the Datadog Agent][3].
+
+To manually add or update rules, load a new `default.policy` file to the Agent. Follow the instructions below:
+
+1. Navigate to the [Rules page][4] in Datadog.
+2. Click **New Rule** button in the top right.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:78]:
```suggestion
2. Click **New Rule** in the top right.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 12]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+3. Select **Workload Security** under rule types.
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:79]:
```suggestion
3. Select **Workload Security** under **Rule types**.
```

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md [hunk 13]
```diff
@@ -68,5 +68,78 @@ For example, if you used explicit filenames to evaluate open events (for example
...
+        enabled: false
+    ```
+
+10. Upgrade the helm chart with the following command: `helm upgrade <RELEASENAME> -f values.yaml --set datadog.apiKey=<APIKEY> datadog/datadog`
+
+    **Note:** If you need to make further changes to `default.policy` you can either use `kubectl edit cm jdefaultpol` or replace the configMap with  `kubectl create configmap jdefaultpol --from-file default.policy -o yaml --dry-run=client | kubectl replace -f -`
+
+{{% /tab %}}
+{{< /tabs >}}
+
```
Comment (alai97) [inline content/en/security_platform/cloud_workload_security/workload_security_rules.md:140]: @sarina-dd do you intend on adding a Further Reading section to this page?

**Review by alai97 — CHANGES_REQUESTED** [review]
Mostly formatting and spelling nits.

File: content/en/security_platform/cloud_workload_security/workload_security_rules.md (hunk 13 above)
Comment (sarina-dd) [reply content/en/security_platform/cloud_workload_security/workload_security_rules.md:140]: Good catch! Let's leave this as-is for now, and once we roll out the new auto-generated CWS rules page, I'll add a further reading section with a link to that page.

**Review by alai97 — APPROVED** [review]
Looks great in the new preview!

**Pull request merged**